app/raid.js

```diff
@@ -187,7 +185,15 @@ class Raid {
 	}
 
 	setMessage(channel, member, raid_id, message) {
-		this.getRaid(channel, member, raid_id).message = message;
+		const raid = this.getRaid(channel, member, raid_id),
+			current_message = raid.message;
+
+		if (current_message) {
+			current_message.delete();
```

Comment:
I didn't want to delete the old message b/c someone could be looking at it, so if every time someone calls status, the message is deleted, and they have to quickly scroll down through the message to find what they were looking at, then they'll miss all the chatter of what people are saying.